python-modules/cis_postgresql/setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
```

Comment:
this no longer works, pip warehouse only support retructured text/rst (not markdown)
one quick fix is to just... not have a description or just a link to our git repos. a better fix would be to convert the text.

Reply:
I'll open an issue to fix separately.